test.js

```diff
@@ -7,6 +7,11 @@ import tempy from 'tempy';
 const bin = path.join(__dirname, 'cli.js');
 const fixture = fs.readFileSync(path.join(__dirname, 'fixtures/code-of-conduct.md'), 'utf8');
 
+test.beforeEach(async () => {
+	const cwd = tempy.directory();
+	await execa(bin, ['--language=en'], {cwd});
+});
```

Comment:
I think this should rather be executed just for the language tests and not all tests. So use a normal function instead of a `beforeEach` hook.